test/Visualization_Tests/src/Table_Visualisation_Integration_Spec.enso

```diff
@@ -0,0 +1,52 @@
+from Standard.Base import all
+
+from Standard.Table import all
+from Standard.Test import all
+import Standard.Visualization.Table.Visualization
+
+## These tests pair with the Playwright tests found at app/gui/integration-test/project-view/tableVisualisation.spec.ts
+   These tests check the json produced by prepare_visualization matches a baseline
+   The Playwright tests check that json data then renders correctly in an AG Grid in the GUI
+   If you change the json API you can regen the reference json by commenting in the line of code in check_equal
+   Then run these tests and then run the js prettier
+   Remember to comment the write back out
+
+json_dir = enso_project.data . parent . parent . parent / 'app/gui/integration-test/project-view/table-vis-json'
+type TestData
+    Value ~data
+
+    t self = self.data
+  
+    setup = TestData.Value <|
+        vector1 = Data.list json_dir '{*.json}'
+        files = Data.read_many vector1 ..Plain_Text
+        values = files.at 'Value' . map x->(x.parse_json.to_text)
+        keys = files.at 'Path' . map x->(x.name.split '.' . at 0)
+        Dictionary.from_keys_and_values keys values
+
+    check_equal self vis filename =
+        ## Comment this line in and run the tests to regen the test data if you change the API
+        ## But don't commit with the next line commented in
+        ## vis.write (json_dir / (filename + '.json')) on_existing_file=..Overwrite
+        vis . should_equal (self.data.at filename)
```

Comment:
I think an easier to use alternative may be to replace this with `if Environment.get "ENSO_TABLE_VISUALIZATION_TEST_WRITE" == "true"`.

The user may just run the tests with the variable set to enable the 'writeback'. IMO this is much easier than modifying the source code and having to remember to then uncomment it again.